changelogs/changelog_2_0_0.md

```diff
@@ -442,7 +442,37 @@
 
 - When compiling for Release the flag `-fno-math-errno` is used for GCC.
 
+## Docgen
+
+- Markdown mode is now default markup language of doc comments (instead
+  of legacy `RstMarkdown` mode), which means that some RST features,
+  that have a Markdown equivalent, are turned off.
```

Comment:
```suggestion
  that do not have a Markdown equivalent, are turned off.
```

?

Reply:
I rewrote/expanded this point of the list.